components/gitpod-cli/cmd/credential-helper.go

```diff
@@ -63,7 +93,8 @@ var credentialHelper = &cobra.Command{
 			}
 
 			cmdLineString := strings.ReplaceAll(cmdLine, string(byte(0)), " ")
-			gitCmdInfo.parseGitCommandAndOriginRemote(cmdLineString)
+			log.Printf("cmdLineString -> %v", cmdLineString)
```

Comment:
Is it just for debugging? Can be removed now?

Can we leak anythign senstive by doing it, i.e. comamnd include some token?